doc/flame/effects.md

```diff
@@ -182,8 +190,16 @@ final effect = ScaleEffect.by(Vector2.all(1.5), EffectController(duration: 0.3))
 
 This effect works similar to `ScaleEffect.by`, but sets the absolute value of the target's scale.
 
+ ```{flutter-app}
+ :sources: ../flame/examples
+ :page: scale_to_effect
+ :show: widget code infobox
+ :width: 180
+ :height: 160
+ ```
+
 ```dart
-final effect = ScaleEffect.to(Vector2.zero(), EffectController(duration: 0.5));
+final effect = ScaleEffect.to(Vector2.all(0.5), EffectController(duration: 0.5));
```

Comment:
Could you please make this code multi-line so that the user won't have to scroll to see it? Something like
```
final effect = ScaleEffect.to(
  Vector2.all(0.5), 
  EffectController(duration: 0.5),
);
```
And similarly on line 185